documentation/mkdocs.yml

```diff
@@ -47,8 +49,6 @@ extra:
       code: 'Ubuntu Mono'
 markdown_extensions:
     - admonition
-    - codehilite(guess_lang=false)
```

Comment:
This should stay. In case this makes trouble locally it must be ensured that this lines does not make the way into master.